### PR TITLE
[WIP] Refactor to chosen license properties card

### PR DESCRIPTION
**Pull request by akmadian**
**Description**
This PR will change the method of displaying the selected license to using a card, with a description of the license, and what the license allows licensees to do RE sharing and adapting. The inspiration for this design can be found in [this Figma](https://www.figma.com/file/BOdtIkXh65XCksKr32ZZHcgE/Creative-Commons-Licensing-Project?node-id=0%3A1)

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by peterspdx** [comment]
Who is the decision maker before this goes live? @kgodey

**Comment by akmadian** [comment]
@peterspdx do you mean this PR, or the project as a whole?

**Comment by brenoferreira** [comment]
Image 03-07-19 at 09 35

Below is a mockup of the proposed changes
Screen Shot 2019-07-03 at 09 43 56
_note that the license description has a slightly smaller font-size (`0.9em`)

**Comment by brenoferreira** [comment]
I also added a few margins between the texts

File: src/index.html [hunk 1]
```diff
@@ -234,33 +242,6 @@ <h4 class="title is-4">
                             </div>
                         </div> <br><br>
```
Comment (brenoferreira) [inline src/index.html]: Don't use `<br>` to add margins or space between elements. Use `margin` or `padding` for that. `<br>` is for line breaks inside paragraphs `<p>` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#Accessibility_concerns

File: src/index.html [hunk 2]
```diff
@@ -234,33 +242,6 @@ <h4 class="title is-4">
...
-                                <div class="column is-one-quarter">
-
-                                </div>
-                                <div class="column">
-                                    <p v-bind:href="chooser.selected_license_link" class="selected-license-name">
-                                    {{chooser.selected_license}} ({{chooser.selected_license_short}})
-                                    </p>
-                                    <p>
-                                        {{chooser.selected_license_description}}
-                                    </p> <br>
```
Comment (brenoferreira) [inline src/index.html]: Same here

**Comment by kgodey** [comment]
@peterspdx before the new license chooser goes live, we'll run it past comms, legal, and product.

**Pull request merged**